apis/openstack-swift/src/main/java/org/jclouds/openstack/swift/v1/blobstore/RegionScopedSwiftBlobStore.java

```diff
@@ -819,10 +817,7 @@ public void run() {
                }
             }
             // Finished writing results to stream
-            try {
-               output.close();
-            } catch (IOException e) {
-            }
+            Closeables2.closeQuietly(output);
```

Comment:
I've just seen a bunch of "System.out" in this class. I know it is not the scope of this PR, but it would be great to replace them by a logger!